assessment_software/src/app/table-component/table.component.ts

```diff
@@ -0,0 +1,22 @@
+import {Component, Input} from '@angular/core';
+
+@Component({selector:'app-table',
+templateUrl:'./table.component.html',
+styleUrls:['./table.component.css']
+})
+
+export class TableComponent{
+    @Input() headers:string[]=[]; 
+    @Input() width:Number=700;
+    @Input() data: {[key: string]: string}[]=[{
+      Name:"",
+      ID:"",
+      Passcode:"",
+      Email:"",
+     }
+   ];
+   @Input() wrapStyle:string='';
+   @Input() scrollStyle:string='';
```

Comment:
it is better to make it wrapping=true/false and scrollable=true/false

Reply:
Done